build.sh

```diff
@@ -17,7 +17,7 @@ build()
     mkdir -p "${BUILD_DIR}"
     cd "${BUILD_DIR}" || return
     echo "Building with cmake"
-    cmake .. -DCMAKE_INSTALL_PREFIX=/usr -DCPACK_PACKAGE_VERSION="${RELEASE_VERSION}" -DCPACK_PACKAGE_NAME="${PACKAGE_NAME}" -DCPACK_PACKAGE_ARCH="${ARCH}"
+    cmake .. -DCMAKE_INSTALL_PREFIX=/usr -DCPACK_PACKAGE_VERSION="${RELEASE_VERSION}" -DCPACK_PACKAGE_NAME="${PACKAGE_NAME}" -DCPACK_PACKAGE_ARCH="${ARCH}" -DEMBEDDED=ON
```

Comment:
I found a comment in deploy_to_printer.sh in Jedi-build:
`# -DEMBEDDED yields warnings for packages that don't support it, only fdm_materials`

Did you check if you get a warning enabling this flag?